screens/Home/MyVcsTab.tsx

```diff
@@ -233,6 +241,16 @@ export const MyVcsTab: React.FC<HomeScreenTabProps> = props => {
         buttonText={t('common:ok')}
         customHeight={'auto'}
       />
+
+      <MessageOverlay
+        isVisible={controller.isDownloadLimitExpires}
+        title={t('errors.downloadLimitExpires.title')}
+        message={downloadFailedVcsErrorMessage}
+        onButtonPress={controller.DELETE_VC}
```

Comment:
As we are deleting VC on Ok button click, so make sure this popup is not dismissed on outside touch/click